src/main/kotlin/org/cascadebot/bot/events/InteractionListener.kt

```diff
@@ -38,14 +38,14 @@ class InteractionListener : ListenerAdapter() {
         val args = CommandArgs(event.options.groupBy { it.name })
 
         if (event.isGlobalCommand) {
-            val command = Main.commandManager.getCommand(event.commandPath)
+            val command = Main.commandManager.getCommand(event.fullCommandName)
             if (command == null) {
                 context.reply("Command not recognised!", MessageType.DANGER)
                 return
             }
             logger.info(
-                "Command {} executed by {} with args: {}",
-                event.commandPath,
+                "Command \"{}\" executed by {} with args: {}",
+                event.fullCommandName,
```

Comment:
I take it this replaces commandPath and instead of being separated by `/` it's now separated by ` `. Seams a bit weird that this was done, but I did look and this is how we do it now so not much we can do (unless we make our own method)

Reply:
Yeah this is a change by JDA as listed under https://jda.wiki/introduction/migration-v4-v5/#naming-changes

Reply:
Words exactly the same thought just with spaces so should be no issues